src/site/markdown/server/manage-webadmin.md

```diff
@@ -2496,6 +2497,45 @@ Response codes:
 
 Not implemented yet.
 
+## Deleted Messages Vault
+
+Deleted messages from an user will be stored in a restricted place where only administrators can access.
```

Comment:
`be stored in a restricted place where only administrators can access` -> is it?

I would suggest: 

```
The 'Deleted Message Vault plugin' allows you to keep users deleted messages during a given retention time. This set of routes allow you to *restore* users deleted messages or export them in an archive (not implemented yet).

To move deleted messages in the vault, you need to specifically configure the DeletedMessageVault PreDeletionHook.
```